src/main/java/com/cen4010/bookstore/book/BookRepository.java

```diff
@@ -1,10 +1,15 @@
 package com.cen4010.bookstore.book;
 
 import java.util.UUID;
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface BookRepository extends JpaRepository<Book, UUID> {
 
+  List<Book> findByGenre(String genre);
+
+  List<Book> findTop10ByOrderByCopiesSoldDesc();
+
```

Comment:
How do these work? Does JPA infer how to create a query based off the name of the method alone?!